### DISCUSSION TITLE
LXQt System Theme

### DISCUSSION THREAD

**Discussion by edwardp3** [lxqt/lxqt#2092] [Q&A] [answered]
The System theme originally did not include the bi-colored main menu icon, but noticed one of the developers included it in the theme posted here. I downloaded the file (mainmenu.svg) from the system theme directory and copied it into /usr/share/lxqt/themes/system. I logged out and logged back in, changed the theme from Debian to System and the icon still does not appear. Am I missing additional files?

Thanks in advance.

![snapshot82](https://user-images.githubusercontent.com/73129450/137366280-0ad8dabf-e6e7-42e1-904f-c9b5bce45984.png)

**Comment by tsujan** [comment, marked as answer]
> Am I missing additional files?

Yes, of course. You need the whole theme.

**Reply by edwardp3** [reply]
The lxqt-themes package from Debian is installed, current version is 0.16.0. I installed Debian using the LXQt install image from (then) Debian 10.

**Reply by stefonarch** [reply]
Copying  lxqt-panel.qss too is needed.
Or just copy the "system" folder from [here](https://github.com/lxqt/lxqt-themes/tree/master/themes) to `~/.local/share/lxqt/themes`

There is an issue with some widgets themes which produce truncated text, see https://github.com/lxqt/lxqt-themes/issues/40

**Reply by edwardp3** [reply]
Thanks for the info. I copied the lxqt-panel.qss file into the .../themes/system directory and it resulted in a black on white icon, not the two-colored version, plus the panel was 100% transparent.

Not quite what I was expecting.  :)

I switched it back to the Debian theme.

**Reply by tsujan** [reply]
> I copied the lxqt-panel.qss file into the .../themes/system directory and it resulted in a black on white icon, not the two-colored version, plus the panel was 100% transparent.

You should have made a mistake somewhere. Anyway, as @stefonarch said, `~/.local/share/lxqt/themes` should be used. Manual changing of files of an installed package may cause various problems (wrong permissions, interfering with upgrades, etc.).

**Reply by edwardp3** [reply]
I put the original lxqt-panel.qss file back into the .local/share/lxqt/themes/system directory and removed the downloaded mainmenu.svg file, so it has been restored to the original Debian-installed state.

Apparently, additional packages are required to be installed in order to download a folder from GitHub, so I'll just leave it as it is. I'm not a developer or maintainer, just and end user.

**Reply by tsujan** [reply]
No one needs to be a developer to download from GitHub. It can be done by any browser: The repository → Code → Download ZIP.

**Reply by edwardp3** [reply]
Slight correction, the original directory where I restored the original 'system' directory, was /usr/share/lxqt/themes. 

Downloaded the .tar file, created ~/.local/share/lxqt, extracted the system folder from the .tar, it created /themes/system in the process and the System theme now displays the icon. System is now listed at the top of the listing of themes in LXQt Appearance.

tsujan - thanks for the extra info and download tip.

**Reply by edwardp3** [reply]
For good measure, I also added the Silver theme in the same manner as System and it looks great! One quirk (although not LXQt-related) is one particular icon that loads in, is white, its color cannot be changed AFAIK and when using the lighter themes, the icon becomes invisible due to the panel being white with those particular themes. 

But all in all, I like LXQt and I'm glad I used the LXQt image to install Debian. My desktops are 10 and 12 years old and still run great.

**Reply by stefonarch** [reply]
Nice to hear!
You can change panel backgound color and transparency.

**Reply by edwardp3** [reply]
HI stefonarch,

Thanks. I found those options and tried some of the tweaks. 

Attached is a screenshot of neofetch. This is on the 12-year old desktop. Aside from LXQt/Linux, Thunderbird was minimized and the terminal window was open. Using 846MB (MiB) of RAM.  :)

![neofetch](https://user-images.githubusercontent.com/73129450/137549209-aee87b8c-1bab-4540-b93a-0c9102d35f5b.png)

**Comment by edwardp3** [comment]
Was successful installing both Lubuntu and PCLinuxOS LXQt images into virtual machines and they work well. Downloaded the themes .tar and copied System and Silver into .local/share/lxqt/themes and they work great, virtually.  :)

![pclinuxos-neofetch](https://user-images.githubusercontent.com/73129450/138343414-4959f082-0026-45c2-b6b5-9c943326b4fc.png)
